lib/filesystem/include/filesystem/filesystem.hpp

```diff
@@ -38,6 +38,15 @@ namespace irods::experimental::filesystem
         no_trash
     };
 
+    struct extended_remove_options
+    {
+        bool no_trash   = false;
+        bool verbose    = false;
+        bool progress   = false;
+        bool recursive  = false;
+        bool unregister = false;
+    };
```

Comment:
are any of these flags mutually exclusive?

Reply:
This is just a personal preference (and maybe this was done to match some existing flag) but I would change `no_trash` to just `trash`, `send_to_trash`, `skip_trash`, or some positive word so as to avoid a double-negative as the default. Fine as-is, just a suggestion

Reply:
agreed on positive word - my other python tool used a 'mode' from ['trash', 'force', 'unregister'], mutually exclusive.